content/en/docs/contribute/style/style-guide.md

```diff
@@ -358,6 +359,70 @@ Shortcodes inside include statements will break the build. You must insert them
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td><ul><li>Use ordered headings to provide a meaningful high-level outline of your content.</li>
+  <li>Use only one first-level heading per page.</li>
+  <li>For non-blog post content, use pound or hash signs (#), not underlines (--- or ===) to designate first-level headings. </li>
+  <li>Use sentence case for headings. For example, **Extend kubectl with plugins** </li></ul></td><td><ul><li>Use first level heading in blog posts, as Hugo will automatically convert the title section of your post’s front matter into a first-level heading.</li>
+  <li>Use headings level 4 through 6, unless it is absolutely necessary. If your content is that detailed, it may need to be broken into separate articles.</li><li>Use title case for headings. For example, **Extend Kubectl With Plugins** </li></ul></td></tr>
+</table>
+
+### Paragraphs
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td><ul><li>Try to keep the paragraphs on the shorter side. 5 to 6 sentences maximum.</li>
+  <li> Use three hyphens (---) to create a horizontal rule. Use horizontal rules for breaks in paragraph content. For example, a change of scene in a story, or a shift of topic within a section.</li></ul></td><td><ul><li>Indent the first paragraph with space characters. For example, ⋅⋅⋅Three spaces before a paragraph will indent it.</li>
+  <li> Use horizontal rules for decoration.</li></ul>
+</td></tr>
+</table>
+
+### Links
+ <table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td><ul><li>Links should provide context for the content they link to. Certain ports are open on your machines. See <a href="#check-required-ports">Check required ports</a> for more details.</li> <li>Use Markdown-style links ([link text](URL)).</li></ul></td><td><ul><li>Use ambiguous terms like “click here”. Certain ports are open on your machines. See <a href="#check-required-ports">here</a> for more details.</li> <li>Use HTML style links (&#60;link href="/media/examples/link-element-example.css" target="_blank"&#62;Visit our tutorial!)</li><li>Create links that open in new tabs or windows. For example, [example website](https://example.com){target="_blank"}</li></ul>
+</td></tr>
+</table>
+
+### Lists
+Grouping related items not just looks good visually, but also helps the non-sighted users. When a screen reader comes across a list—whether it is an ordered or unordered list—it will be announced to the user that there is a group of list items. The user can then use the arrow keys to cursor up and down between the various items in the list.
+Website navigation links can also be marked up as list items; after all they are nothing but a group of related links.
+
+ - Put a period at the end of every list item. 
+ - Uses dashes (-) for unordered lists. 
```

Comment:
Verifying: Do we use the - or the *? Just checking because I have come across the * so far :)

Reply:
@shavidissa Thanks, yes * also works :)